doc/source/parameters/unresolved_cfd-dem/cfd_dem.rst

```diff
@@ -0,0 +1,28 @@
+***********************************************
+CFD-DEM
+***********************************************
+This subsection includes parameters related to multiphase flow simulations using the both the gls_vans solver and the cfd-dem solver within Lethe.
+
+.. code-block:: text
+
+subsection cfd-dem
+   set grad div = false
+   set void fraction time derivative = false
+   set drag force = true
+   set buoyancy force = true
+   set shear force = true
+   set pressure force = true
+   set drag model = dallavalle
+   set post processing = true
+   set coupling frequency = 200
+   set inlet boundary id = 2
+   set outlet boundary id = 3
+end
+
+* The `grad div` parameter allows the enabling of the grad div stabilization for the Volume Averaged Navier Stokes equations. This allows for a much better mass conservation of the system.
+* The `void fraction time derivative` parameters allows us to choose whether or not we want to account for the time derivative of the void fraction or take it equal to zero.
```

Comment:
According to the convention used in the documentation, the parameters are written using `` ``, you have used ` `.
This should be addressed for all the parameters.